modules/namespace-exclusions.adoc

```diff
@@ -6,8 +6,39 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="namespace-exclusions_{context}"]
-= Namespace exclusions
+= Namespace exclusions from admission controller enforcement
 
-By default, {product-title-short} excludes certain administrative namespaces, such as the `stackrox`, `kube-system`, and `istio-system` namespaces, from enforcement blocking. The reason for this is that some items in these namespaces must be deployed for {product-title-short} to work correctly.
+[role="_abstract"]
+By default, {rh-rhacs-first} excludes certain administrative namespaces from admission controller enforcement blocking. The reason for this is that some items in these namespaces must be deployed for {product-title-short} to work correctly. 
+
+These exclusions mean that API calls for these namespaces are not passed to the admission controller for review. Instead of rejecting the `CREATE` verb, {product-title-short} allows the deployment to be created, but then it is scaled down using Sensor. 
```

Comment:
Suggested removal: Beginning from Instead of ....

Also given my comment above, the line "These exclusions ..." could also be deleted